infrastructure/src/main/java/org/corfudb/infrastructure/NettyServerRouter.java

```diff
@@ -76,7 +76,9 @@ public NettyServerRouter(ImmutableList<AbstractServer> servers, ServerContext se
             try {
                 server.getHandler().getHandledTypes().forEach(handledType -> handlerMap.put(handledType, server));
             } catch (UnsupportedOperationException ex) {
-                log.error("No registered CorfuMsg handler for server {}", server, ex);
+                if (log.isTraceEnabled()) {
```

Comment:
This was originally from one of Maithem's comments, (https://github.com/CorfuDB/CorfuDB/pull/2838#discussion_r542945405). I'm a little inclined to keep current way to make it consistent, and since this happens in a lot of places, it might be worth adding this check?

Reply:
@xcchang In Maithem's comment, I think the concern was the computation of `TextFormat.shortDebugString`. In this particular instance, we don't have this problem.

Reply:
Oh I see now, thanks!